Expensify/Bedrock-PHP
# Add commitCount to all updateJob and createJob data

Needed to fix this specific instance  https://github.com/Expensify/Expensify/issues/88302#issuecomment-429371913

Hold on https://github.com/Expensify/Bedrock/pull/514

Tests:
- Create a job without having done any bedrock call in any cluster. Check it created a job without adding any commitCounts.
- Create a job having done a call in only one cluster. Check it created the job and commitCounts contains only the count for that one cluster.
- Create a job having done a call in more than one cluster. Check it created the job and commitCounts contains all the counts for the clusters.
- Run BWM on the jobs created above and check that they correctly pass the commitCount (note: the webrock commitCount is updated again [here](https://github.com/Expensify/Bedrock-PHP/blob/1236fe20c4c3b82cbb50a56d67d918733edd765f/bin/BedrockWorkerManager.php#L287), which is correct given that we want the commitCount of the GetJobs to be included, which will always be greater than the commitCount for when the job was queued).
